src/main/kotlin/no/nav/familie/tilbake/oppgave/OppdaterOppgaveTask.kt

```diff
@@ -32,13 +34,39 @@ class OppdaterOppgaveTask(
 
         val frist = task.metadata.getProperty("frist")
         val beskrivelse = task.metadata.getProperty("beskrivelse")
-        val saksbehandler = task.metadata.getProperty("saksbehandler")
+        val saksbehandler = task.metadata.getProperty("saksbehandler").takeIf { saksbehandler ->
+            saksbehandler.isNotBlank() && saksbehandler != Constants.BRUKER_ID_VEDTAKSLØSNINGEN
+        }
         val behandlingId = UUID.fromString(task.payload)
+        val enhet = task.metadata.getProperty("enhet")
 
-        val oppgave = oppgaveService.finnOppgaveForBehandlingUtenOppgaveType(behandlingId)
+        val oppgave = try {
+            oppgaveService.finnOppgaveForBehandlingUtenOppgaveType(behandlingId)
+        } catch (e: ManglerOppgaveFeil) {
+            log.warn("Fant ingen oppgave å oppdatere på behandling $behandlingId. Vil forsøke å opprette en ny isteden")
```

Comment:
kunne vi kanskje ha logget dette som error som ett sikkerhetsnett hvis vi oppretter mange oppgaver ved feil.

Eventuellt må vi følge med manuellt i en periode for å se hvordan dette går.